dev-util/idea-community/idea_community_bin-2025.1.4.1.recipe

```diff
@@ -4,13 +4,13 @@ software."
 HOMEPAGE="https://www.jetbrains.com/idea/"
 COPYRIGHT="2025 JetBrains s.r.o."
 LICENSE="Apache v2"
-REVISION="1"
+REVISION="2"
 buildVersion="251.27812.49"
 SOURCE_URI="https://download.jetbrains.com/idea/ideaIC-$portVersion.tar.gz#noarchive"
 CHECKSUM_SHA256="34e0d7d20b6ff303cfbb97c75147b11a437221b96783687d9909adf1d56817ff"
 ADDITIONAL_FILES="IntelliJ_IDEA_Logo.hvif"
 
-ARCHITECTURES="any"
+ARCHITECTURES="all !x86_gcc2"
```

Comment:
Another idea: a bit of a hack, but should work:

```suggestion
if [ "$targetArchitecture" != "x86_gcc2" ]; then
	ARCHITECTURES="any"
else
	ARCHITECTURES="!any"
fi
```
and the inverse in the other recipe.

We don't deduplicate "any" packages yet.